### PR TITLE
CI Update - .NET 7 Preview 1

### PR DESCRIPTION
**Pull request by gewarren**
Update .NET API docs for .NET 7 Preview 1.

This update uses the experimental System.Runtime.dll in the shared framework moniker (.NET 7).

Some of the updates were broken out into smaller commits for review.

### CODE REVIEW

File: xml/System.Globalization/GregorianCalendar.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <Type Name="GregorianCalendar" FullName="System.Globalization.GregorianCalendar">
   <TypeSignature Language="C#" Value="public class GregorianCalendar : System.Globalization.Calendar" />
-  <TypeSignature Language="ILAsm" Value=".class public auto ansi beforefieldinit GregorianCalendar extends System.Globalization.Calendar" FrameworkAlternate="dotnet-uwp-10.0;net-5.0;net-6.0;netcore-1.0;netcore-1.1;netcore-2.0;netcore-2.1;netcore-2.2;netcore-3.0;netcore-3.1;netstandard-1.3;netstandard-1.4;netstandard-1.6;netstandard-2.0;netstandard-2.1" />
+  <TypeSignature Language="ILAsm" Value=".class public auto ansi beforefieldinit GregorianCalendar extends System.Globalization.Calendar" FrameworkAlternate="dotnet-uwp-10.0;net-5.0;net-6.0;net-7.0;netcore-1.0;netcore-1.1;netcore-2.0;netcore-2.1;netcore-2.2;netcore-3.0;netcore-3.1;netstandard-1.3;netstandard-1.4;netstandard-1.6;netstandard-2.0;netstandard-2.1" />
```
Comment (carlossanlop) [inline xml/System.Globalization/GregorianCalendar.xml:3]: Seems it worked!👌

**Review by carlossanlop — APPROVED** [review]
I'm unable to see System.Security.Cryptography. How does it look? Did mdoc generate files for that namespace/assembly?

**Comment by gewarren** [comment]
> I'm unable to see System.Security.Cryptography. How does it look? Did mdoc generate files for that namespace/assembly?

Yes, it looks correct, e.g. [AsnEncodedData](https://github.com/gewarren/dotnet-api-docs/blob/net7-preview1-take5/xml/System.Security.Cryptography/AsnEncodedData.xml). You can also see it in the framework index for .NET 7 [here](https://raw.githubusercontent.com/gewarren/dotnet-api-docs/net7-preview1-take5/xml/FrameworksIndex/net-7.0.xml).

**Comment by carlossanlop** [comment]
Are the CI warnings ok to ignore?

**Comment by gewarren** [comment]
> Are the CI warnings ok to ignore?

The remarks on enums suggestions are okay to ignore. For the inheritdoc suggestions, I've logged https://dev.azure.com/ceapex/Engineering/_workitems/edit/567948.

**Pull request merged**